src/backend/dx12/src/command.rs

```diff
@@ -368,25 +357,50 @@ impl command::RawCommandBuffer<Backend> for CommandBuffer {
         }
     }
 
-    fn clear_color(
+    fn clear_color_image(
         &mut self,
-        rtv: &n::RenderTargetView,
+        image: &n::Image,
         _: image::ImageLayout,
-        color: ClearColor,
+        range: image::SubresourceRange,
+        value: com::ClearColor,
     ) {
-        self.clear_render_target_view(rtv.handle, color, &[]);
+        assert!((image::ASPECT_COLOR).contains(range.aspects));
```

Comment:
Can we use `assert_eq!` here as it **must** be `ASPECT_COLOR

Reply:
I tried to carefully support the case where the mask is empty

Reply:
The aspect mask **must** not be empty for a subresource range.